src/helpers/codetohtmlconverter.cpp

```diff
@@ -49,7 +49,8 @@ void CodeToHtmlConverter::initCodeLangs() Q_DECL_NOTHROW {
         {QStringLiteral("vex"), CodeToHtmlConverter::CodeVex},
         {QStringLiteral("xml"), CodeToHtmlConverter::CodeXML},
         {QStringLiteral("yml"), CodeToHtmlConverter::CodeYAML},
-        {QStringLiteral("yaml"), CodeToHtmlConverter::CodeYAML}};
+        {QStringLiteral("yaml"), CodeToHtmlConverter::CodeYAML},
+        {QStringLiteral("yaml"), CodeToHtmlConverter::CodeForth}};
```

Comment:
Shouldn't that be something like `forth`?

Reply:
You are right slight miss on my side. Sorry Will update quickly.